library/Notifications/Widget/ItemList/ObjectLabelList.php

```diff
@@ -1,17 +0,0 @@
-<?php
-
-/* Icinga Notifications Web | (c) 2023 Icinga GmbH | GPLv2 */
-
-namespace Icinga\Module\Notifications\Widget\ItemList;
-
-use Icinga\Module\Notifications\Common\BaseItemList;
-
-class ObjectLabelList extends BaseItemList
-{
-    protected $defaultAttributes = ['class' => ['minimal', 'incident-object-label-list']];
```

Comment:
The class `incident-object-label-list` is still referenced in css. Please open a followup PR removing that.

Reply:
It has been removed in this PR #130